000-local-env.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+if ( defined( 'VIP_GO_ENV' ) && VIP_GO_ENV ) {
```

Comment:
> I think we should still use WPCOM_IS_VIP_ENV for this check

We'd also have to fix `WPCOM_IS_VIP_ENV` because it currently always returns true. It's flagged as deprecated and only defined for back-compat, so I think this was on purpose.

> Currently VIP_GO_ENV is set to false in dev, but IMO it should be set to something like local-development.

Hmm... I think this is right. The idea is that it returns false if it's not a VIP Go Environment.

> The environment name constant specifies which 'version' of the site the code is running on, vs. where the code is physically running (Go vs local).

It really just specifies the environment name or `false` if it's not a VIP Go environment.